### ISSUE TITLE
error sorting in typus

### ISSUE THREAD

**Issue by phoet** [phoet/on_ruby#185]
@kikito did you see those errors? they are hopefully fixed via 38e0abe6c1c647e9e0744267e09c4fa221443ca2

```
An ActionView::Template::Error occurred in materials#index:

PG::AmbiguousColumn: ERROR: column reference "id" is ambiguous
LINE 1: ...id" WHERE "events"."label" = 'madridrb' ORDER BY id asc LIM...
^
: SELECT "materials"."id" AS t0_r0, "materials"."name" AS t0_r1, "materials"."description" AS t0_r2, "materials"."url" AS t0_r3, "materials"."user_id" AS t0_r4, "materials"."event_id" AS t0_r5, "materials"."created_at" AS t0_r6, "materials"."updated_at" AS t0_r7, "materials"."preview_type" AS t0_r8, "materials"."preview_code" AS t0_r9, "users"."id" AS t1_r0, "users"."nickname" AS t1_r1, "users"."name" AS t1_r2, "users"."image" AS t1_r3, "users"."url" AS t1_r4, "users"."location" AS t1_r5, "users"."description" AS t1_r6, "users"."created_at" AS t1_r7, "users"."updated_at" AS t1_r8, "users"."github" AS t1_r9, "users"."admin" AS t1_r10, "users"."freelancer" AS t1_r11, "users"."available" AS t1_r12, "users"."slug" AS t1_r13, "users"."hide_jobs" AS t1_r14, "users"."twitter" AS t1_r15, "users"."email" AS t1_r16, "users"."super_admin" AS t1_r17, "events"."id" AS t2_r0, "events"."name" AS t2_r1, "events"."date" AS t2_r2, "events"."description" AS t2_r3, "events"."location_id" AS t2_
r4, "events"."user_id" AS t2_r5, "events"."created_at" AS t2_r6, "events"."updated_at" AS t2_r7, "events"."published" AS t2_r8, "events"."slug" AS t2_r9, "events"."label" AS t2_r10, "events"."limit" AS t2_r11 FROM "materials" INNER JOIN "events" ON "events"."id" = "materials"."event_id" AND "events"."label" = $1 LEFT OUTER JOIN "users" ON "users"."id" = "materials"."user_id" WHERE "events"."label" = 'madridrb' ORDER BY id asc LIMIT 20 OFFSET 0
app/middlewares/cookie_domain.rb:11:in `call'


-------------------------------
Request:
-------------------------------

* URL : http://www.madridrb.com/admin/materials?order_by=id&sort_order=asc
* HTTP Method: GET
* IP address : 194.224.214.178
* Parameters : {"order_by"=>"id", "sort_order"=>"asc", "controller"=>"admin/materials", "action"=>"index"}
* Timestamp : 2015-10-15 12:12:35 +0200
* Server : bf654239-ee92-4339-8d46-f91ee23c704f
* Rails root : /app
* Process: 19

----------------------------
Backtrace:
-------------------------------

app/middlewares/cookie_domain.rb:11:in `call'
```

**Comment by kikito** [comment]
Hi there,

Sorry I didn't answer to the previous issue - I didn't see it.

The website seemed to work just fine for me before, but *now* (probably after the update) it seems there is a problem.

In the administration page, I have lost the "select by typing" feature in some fields, and they have been replaced by "plain html selects", which are basically unusable (the results are not even ordered alphabetically)

The selects appear as soon as I click on the input field - I can't type anything now. Screenshot:

![screen shot 2015-10-24 at 12 38 12](https://cloud.githubusercontent.com/assets/63131/10710160/42312df2-7a4c-11e5-86d6-82851e3e5d34.png)

The javascript console shows the following error, maybe it is related:

```
Uncaught ReferenceError: AbstractChosen is not defined
```

**Comment by phoet** [comment]
i hope if fixed this through 7c9e7c1bee517acf3f87e5ca54c1654c75907808 

chosen versions seemed to be incompatible.

**Comment by kikito** [comment]
Hi, this seems to be fixed now. Thanks for the fast response!
